answers/os.md

```diff
@@ -84,7 +84,18 @@ Child process cannot change variables of its parent.
 
 ## Concurrency vs Parallels? (in case single CPU core and multiple CPU cores)
 
-To be defined.
+Concurrency: executing multiple tasks at the same time but not necessarily simultaneously.
+
+Parallelism: means that an application splits its tasks up into smaller subtasks which can be processed in parallel, for instance on multiple CPUs at the exact same time.
```

Comment:
In my opinion, `application`, `subtasks` and `processed` are not the right choices of words. We are talking about OS, where words like applicaion, process, task have their own specific meaning.
And `in parallel` causes confusion also. It's like one is asked to distinguish between red and blue, and his answer is "red is red, while blue is blue".